core/tauri-runtime-wry/src/system_tray.rs

```diff
@@ -105,7 +105,9 @@ pub fn create_tray<T>(
 
   #[cfg(target_os = "macos")]
   {
-    builder = builder.with_icon_as_template(system_tray.icon_as_template)
+    builder = builder
+      .with_icon_as_template(system_tray.icon_as_template)
+      .with_icon_as_template(system_tray.menu_on_left_click)
```

Comment:
```suggestion
      .with_menu_on_left_click(system_tray.menu_on_left_click)
```

Reply:
wow copy/pasting is hard for me now 😂, thanks